master/buildbot/reporters/telegram.py

```diff
@@ -262,7 +263,8 @@ def command_LIST(self, args, **kwargs):
                     return
 
             else:
-                changes = yield self.master.db.changes.getRecentChanges(num)
+                rs = resultspec.ResultSpec(order=['-changeid'], limit=num)
```

Comment:
we should use data api instead of db api in reporters.

Reply:
I assume you mean something like
```suggestion
                changes = yield self.master.data.get(('changes',), order=['-changeid'], limit=num)
```

However, if changing like I suggested, the test_reporters_telegram fails since change['when_timestamp'] is an integer and cannot be converted to a pretty date at line274, change['date'] = change['when_timestamp'].strftime('%Y-%m-%d %H:%M').

This happens due to the conversion from data/changes.py line 37.

What would be the best approach here?

Reply:
@tardyp any thoughts?

Reply:
hi @vladbogo you can use ```from buildbot.util import epoch2datetime``` in order to convert it back to timestamp.